plugins/playlist/src/components/PlaylistCard/PlaylistCard.tsx

```diff
@@ -108,7 +113,9 @@ export const PlaylistCard = ({ playlist }: PlaylistCardProps) => {
           <Typography variant="body2" className={classes.label}>
             Description
           </Typography>
-          <MarkdownContent content={playlist.description ?? ''} />
+          <Box className={classes.descriptionBox}>
+            <MarkdownContent content={playlist.description ?? ''} />
+          </Box>
```

Comment:
Hmm, not sure about this fix tbh, perhaps it would be better to truncate the content and having the user click through to see the full description? It can be really annoying to be scrolling down a page that have a lot of mini scroll surface that you keep hitting 😅

Reply:
@kuangp Can you please suggest anything?

Reply:
hm Patrik makes a good point about the UI - perhaps you can find a way to make truncation/ellipsis styling work with `MarkdownContent`? Looks like that is broken right now which is resulting in the description being clipped

Reply:
@kuangp  i have idea like If the description is longer than the limit, we append an ellipsis and provide a “Read More” link. Is that ok?

Reply:
I don't think we need a "Read More" link - as Partrik suggested, the user can click through to see the full description if they want. But, yes, I think we would like to have it truncated with a ellipsis - ideally this would be responsive to the dimensions of the card and your screen size instead of being triggered by a hard limit